contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -1,5 +1,5 @@
 <frontmatter>
-  title: VueJs
+  title: Vue.js
```

Comment:
Perhaps can call it as `Vue`? In the React chapter, it refers to VueJs as `Vue`. Better to standardise.

Reply:
@damithc Is it okay to have the chapter as Vue? Will proceed accordingly then

Reply:
No objections from me.